src/main/java/mate/academy/service/UserService.java

```diff
@@ -15,6 +15,11 @@
      * Return <code>null</code> if there is no suitable user
      */
     public User findByEmail(String email) {
+        for(User u : users){
```

Comment:
```suggestion
        for(User user : users){
```
let's better not use shortened names for variables